pkgs/by-name/le/lesspipe/package.nix

```diff
@@ -85,89 +85,84 @@ stdenv.mkDerivation rec {
       ];
       fake = {
         # script guards usage behind has_cmd test function, it's safe to leave these external and optional
-        external =
```

Comment:
This is nothing you can fix but this behaviour by the new formatter is really dumb.